homeassistant/components/nextdns/coordinator.py

```diff
@@ -40,14 +45,14 @@ class NextDnsUpdateCoordinator[CoordinatorDataT: NextDnsData](
     """Class to manage fetching NextDNS data API."""
 
     config_entry: NextDnsConfigEntry
+    _update_interval: timedelta | None = None
```

Comment:
```suggestion
    _update_interval: timedelta
```
We don't have one without one right